pkg/util/util.go

```diff
@@ -258,3 +260,14 @@ func GenerateSecureRandomString(n int) (string, error) {
 
 	return string(ret), nil
 }
+
+func EnsureKubeVirtGroupVersionKind(obj runtime.Object) (runtime.Object, error) {
```

Comment:
Can you add some reference why this is needed in the first place?

Reply:
ACK done.